src/FormControl/test/FormControlSpec.tsx

```diff
@@ -1,16 +1,22 @@
 import React, { useState } from 'react';
 import { render, fireEvent, screen, waitFor } from '@testing-library/react';
 import sinon from 'sinon';
-import { getDOMNode } from '@test/utils';
 import Form, { FormInstance } from '../../Form';
 import FormControl from '../FormControl';
 import FormGroup from '../../FormGroup';
 import Schema from '../../Schema';
 import Toggle from '../../Toggle';
 import Button from '@/Button';
 import TagInput from '@/TagInput';
+import { testStandardProps } from '@test/utils';
 
 describe('FormControl', () => {
+  testStandardProps(
+    <Form>
+      <FormControl name="username" />
+    </Form>
+  );
```

Comment:
`testStandardProps` will only test the Form, not the FormControl

Reply:
Got it. I made the changes, but I didn't find a way to test it against "custom className prefix", that single one isn't passing. Advice? Maybe I don't know something..

Reply:
get element by [test id](https://github.com/rsuite/rsuite/blob/main/src/FormControl/FormControl.tsx#L191)

```tsx
it('Should have a custom className prefix', () => {
  render(
    <Form>
      <FormControl classPrefix="custom-prefix" name="username" />
    </Form>
  );

  expect(screen.getByTestId('form-control-wrapper')).to.have.class('rs-custom-prefix');
});

```

Reply:
Hmm, ok, but what about testStandardProps()? It keeps testing the prefix. Is there a way to avoid it? Or should I remove it?
I'm sending this options: 
`{
renderOptions: {wrapper: Form},
getRootElement: ()=>screen.getByTestId("element")
}`
It works for the two first cases of testStandardProps(), but not for prefix test.
Am I in the correct way?

Reply:
testStandardProps cannot meet the test of FormControl at present, unless the testStandardProps is modified to add this kind of test for components with wrapper. You can remove `testStandardProps` directly here.